.github/workflows/CI.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches: [master]
+    tags: ['*']
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '0.7'
```

Comment:
Even if this works, do we really want to keep it? My instinct is to only support the most recent LTS and current version unless there's some *really* compelling reason. If for no other reason than to avoid wasting energy on every CI run

Reply:
If it's kept, we can track the minimum Julia dependency. I think these sorts of things should be dropped passively on an as-needs basis when new features are adopted.